packages/dlm/dlm.0.3.3/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "xen-api@list.xensource.com"
+authors: ["Edwin Török"]
+homepage: "https://github.com/xapi-project/ocaml-dlm/"
+bug-reports: "https://github.com/xapi-project/ocaml-dlm/issues"
+dev-repo: "git+https://github.com/xapi-project/ocaml-dlm.git"
+doc: "https://xapi-project.github.io/ocaml-dlm/doc"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "-p" name "-j" jobs "@runtest"] {with-test}
+  ["dune" "build" "-p" name "-j" jobs "@doc"] {with-doc}
+]
+depends: [
+  "ocaml"
+  "conf-pkg-config" {build}
+  "dune" {>= "1.4"}
+  "ctypes" {>= "0.11.1"}
+  "lwt" {>= "4.2.0"}
+  "odoc" {with-doc}
+  "ounit" {>= "2.2.0"}
+  "unix-errno" {>= "0.5.2"}
+  "uuidm" {>= "0.9.6"}
+]
+depexts: [
+  ["libdlm-dev"] {os-distribution = "debian"}
+  ["libdlm-dev"] {os-distribution = "ubuntu"}
+  ["dlm-devel"] {os-distribution = "rhel"}
+  ["dlm-devel"] {os-distribution = "centos"}
+  ["dlm-devel"] {os-distribution = "fedora"}
+  ["dlm-devel"] {os-family = "oraclelinux"}
```

Comment:
```suggestion
  ["dlm-devel"] {os-family = "ol"}
```

Reply:
The values are documented here: https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values